modules/oe_paragraphs_map/README.md

```diff
@@ -0,0 +1,7 @@
+OpenEuropa Paragraphs Map
+==============================
+
+This module provides the Map paragraph that depends on the [OpenEuropa Paragraphs Media Field Storage](https://github.com/openeuropa/oe_content/tree/master/modules/oe_paragraphs_media_field_storage)
+component.
+
+The paragraph displays Webtools map entities.
```

Comment:
We need to add here what contrib modules we need to require in order to have this working, I see  json_field is added to require-dev, we should add that here too. See the event [readme here](https://github.com/openeuropa/oe_theme/blob/EPIC-Event/modules/oe_theme_content_event/README.md) as an example. Also, if we need some permission configuration we should add them here.